microsoft/PowerToys
# [modules] Unify exe & dll naming

## Summary of the Pull Request

**What is this about:**
Prefix all modules dlls and exes with PowerToys. Also prefix all windows shown in task manager with PowerToys.
![image](https://user-images.githubusercontent.com/57057282/145192246-7ef17c18-28fb-41fa-b5df-41d9be19a44d.png)

@crutkas Because of the second last commit (*resources.dll files) it would be good if we could run a test release build on this branch. resources.dll files *should* follow .exe (not project) names, but I'm not 100% sure and can't test this locally as resource.dlls are only generated in pipeline builds. Also, I wanna test it in detail.

Note: I didn't touch PT Run plugins.

**What is include in the PR:** 

**How does someone test / validate:** 
Install PT and confirm that all modules are working.
Confirm that all PowerToys processes in task manager have `PowerToys.` prefix
Confirm that all PowerToys dlls and exes in `Program Files/PowerToys/modules` have `PowerToys.` prefix

## Quality Checklist

- [X] **Linked issue:** #2125 

## Contributor License Agreement (CLA)
A CLA must be signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/PowerToys) and sign the CLA.
